src/main/java/dev/blaauwendraad/masker/json/ValueMaskers.java

```diff
@@ -426,7 +427,7 @@ the byte encoded unicode character (\\u + 4 hex bytes) to prevent possible Array
                                                 decodedBytes[decodedIndex++] = (byte) (0x80 | (unicodeHexBytesAsChar & 0x3f));
                                             }
                                         } catch (IllegalArgumentException e) {
-                                            throw context.invalidJson(e.getMessage(), valueStartIndex);
+                                            throw context.invalidJson(Objects.requireNonNull(e.getMessage()), valueStartIndex);
```

Comment:
How can `e.getMessage()` ever be `null` ?

Reply:
Technically, the exception message can be null, which is why it complains. But in this particular case, we're the ones throwing IAE, so it's safe to assert it

Reply:
Ugh, bit annoying imo to change code just to get the stupid analyser happy, but sure

Reply:
🤷

Reply:
At least that's an exceptional situation :)